### ISSUE TITLE
New metric: AUPRC

### ISSUE THREAD

**Issue by lazappi** [theislab/atlas-feature-selection-benchmark#132]
Thanks for taking the time to suggest a new metric!

## Description

> _Please briefly describe the suggested metric: What is it? What does it measure? Why would it be a good fit for the project?_

Add the Area Under the Precision Recall Curve classification metric

## Links

> _Links to information about the metric_

- **Package:** https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html

## Anything else?

> _Anything else you think is important about the metric_
